crowdsource/tests/test_forms.py

```diff
@@ -0,0 +1,136 @@
+from django.test import TestCase
+from crowdsource.issues import IssueType
+from crowdsource.forms import *
+from opencivicdata.core.models import Jurisdiction, Person, Division
+
+class BaseFormTestCase(TestCase):
+
+    def setUp(self):
+        division = Division.objects.create(
+            id='ocd-division/country:us', name='USA')
+        self.jur = Jurisdiction.objects.create(
+            id="ocd-division/country:us/state:mo",
+            name="Missouri State Senate",
+            url="http://www.senate.mo.gov",
+            division=division,
+        )
+        self.person = Person.objects.create(name='Bikram Bharti')
+
+class IssueFormTestCase(BaseFormTestCase):
```

Comment:
these tests seem to be more about testing if Django's form validation works than if the functionality works, I'd suggest focusing the tests on the code we wrote

Reply:
This does appear to be like that right now, but I wrote them keeping in mind that, I'll have custom fields if I want to carry out integration of form. So these tests will look much better if the fields, I have added directly would have been custom.